src/app/lib/config/optimizely/index.js

```diff
@@ -1,11 +1,18 @@
 export default {
   // This is the key of the whole experiment
-  flagKey: 'newswb_ws_datarecs',
+  flagKey: 'newswb_ws_litesitepromotion',
   // This is the key for the 'rule' that is attached to the experiment
-  ruleKey: 'newswb_ws_datarecs_experiment',
+  ruleKey: 'newswb_ws_litesitepromotion',
   viewClickAttributeId: 'wsoj',
   variationMappings: {
-    on: 'on',
-    off: 'off',
+    control_text_only: 'control_text_only',
+    variation_a_explore_data_friendly_version:
+      'variation_a_explore_data_friendly_version',
+    variation_b_data_saving_version: 'variation_b_data_saving_version',
+    variation_c_read_data_saving_version:
+      'variation_c_read_data_saving_version',
+    variation_d_lite_site: 'variation_d_lite_site',
+    variation_e_2g_optimised_version: 'variation_e_2g_optimised_version',
+    variation_f_low_data_version: 'variation_f_low_data_version',
```

Comment:
Do we need the string values?

This should work (I'm surprised it wasn't caught by linting...)

```suggestion
  variationMappings: {
    on,
    control_text_only
    ...
    } 
```

Reply:
Hello, this one doesn't quite work: 
![Screenshot 2025-03-31 at 11 11 31](https://github.com/user-attachments/assets/044cb121-6d28-4156-9e87-ee093b9fddb4)

Reply:
Oh 😿 we can leave it as is. 

Ah, you know why I thought it would work like this: https://github.com/bbc/simorgh/blob/latest/src/app/components/MediaLoader/configs/index.ts#L26-L37 - the imported function had the same name as the object key. 

My apologies!